build.gradle.kts

```diff
@@ -52,7 +52,7 @@ task<Test>("cleanUpAfterBamboo").apply {
 
 dependencies {
     testCompile(project(":virtual-users"))
-    compile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
+    testCompile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
```

Comment:
What is the difference between compile and testCompile?

Reply:
`compile` is available for `src/main` and `src/test`, while `testCompile` is for `src/test` only
We don't even have `src/main` in this repo, so it doesn't really matter. However, it's a good practice to limit scopes as much as possible.